### PR TITLE
🔧(prettier) add default rules

### PR DESCRIPTION
**Pull request by jbpenrath**
## Purpose
In my VS Code configuration, Prettier auto format code on save. But when I would like to modify my first django template, there was an exception due to Prettier added line break at the wrong places.

Furthermore, to avoid other surprise like these and to homogenize coding style, I just add few rules in the prettierrc based on [Airbnb](https://github.com/airbnb/javascript) and [Google](https://google.github.io/styleguide/jsguide.html) JS Style guides:
- 100 characters per line
- 2 spaces indentation
- Single quote

## Proposal
- [x] Homogenize richie prettier rules
- [x] Prettier must ignore django html templates

### CODE REVIEW

**Comment by lunika** [comment]
Not having `.yml` extension is valid as explain in the documentation https://prettier.io/docs/en/configuration.html

**Comment by jbpenrath** [comment]
> Not having `.yml` extension is valid as explain in the documentation https://prettier.io/docs/en/configuration.html
> 
> > I also encount an error if the .prettierrc does not have .yml extension
> 
> What is this error ?

Without explicit extension, my prettier extension try to use json parser. Yet, I agree, the doc is clear about that, we could use json or yml without explicit extension. I searched in my VS Code configuration if I can set the parser somewhere but I did'nt found something interesting.

File: .prettierrc.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+printWidth: 80
```
Comment (lunika) [inline .prettierrc.yml:1]: In your commit message you suggest 100:

> I propose to use the Airbnb standard which recommends to use 100 characters per line.

Comment (jbpenrath) [reply .prettierrc.yml:1]: Honestly, I prefer 100 characters per line, 80 characters is a little bit strict sometimes. But I saw that in the richie codebase, the rule of 80 characters is used so I change it afterwards.

**Comment by lunika** [comment]
You should merge your two commits IMO and keep the first one

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Because django templates requires syntax which can't contain line break
+# Prettier must ignore all Django templates files
+src/richie/**/*.html
```
Comment (lunika) [inline .prettierignore:3]: Add a new line at the end of the file please

File: .prettierignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Because django templates requires syntax which can't contain line break
```
Comment (lunika) [inline .prettierignore:1]: In your commit message:

`s/s it recommended in python to respect line width of 79 characters,/It is recommened in Python to respect a line width of 79 characters,/`
`s/I propose to use the Airbnb standard/I suggest to use the Airbnb standard/`

**Review by lunika — APPROVED** [review]

**Pull request merged**